src/config.toml

```diff
@@ -0,0 +1,11 @@
+# This is a sample TOML configuration file
+input = "./src"
```

Comment:
Could you please change the path for this to `./examples`.

Reply:
Resolved in commit [8371c9eb](https://github.com/sdthaker/Node-TILify/pull/13/commits/21441eaecd095e242fb3dc277e10194bf512c3e2)